cdap-app-fabric/src/test/java/io/cdap/cdap/support/services/SupportBundleServiceTest.java

```diff
@@ -134,32 +137,21 @@ public void testDeleteOldBundle() throws Exception {
     File tempFolder = new File(configuration.get(Constants.SupportBundle.LOCAL_DATA_DIR));
```

Comment:
nit: Consider renaming to testCleanupFailedBundles instead to better reflect what the unit test is checking for. Thanks!